tests/integration/cattletest/core/test_lb_config.py

```diff
@@ -234,44 +216,20 @@ def test_lb_config_set_listeners(admin_client, super_client):
     # set 2 listeners
     config = config.setlisteners(
         loadBalancerListenerIds=[listener1.id, listener2.id])
-    config = admin_client.wait_success(config)
     assert config.state == 'active'
```

Comment:
Is assert config.state == 'active' necessary?  I don't think setListeners(...) changes it right?